src/redist/targets/GenerateMSIs.targets

```diff
@@ -166,6 +166,7 @@
                       '$(SdkMSIInstallerFile)' ^
                       '$(WixRoot)' ^
                       '$(ToolsetBrandName)' ^
+                      '$(VersionPrefix)' ^
```

Comment:
I hate the way we pipe from MSBuild targets to PowerShell to calls to the various WiX tools.

Eventually I hope we can get rid of the PowerShell scripts and just have tasks that call the WiX tools.

Reply:
or just .wixproj files that call them for us. That's how we did it in aspnet.